packages/test-cases/cases/cli/build-cmd/wasm/typescript/001-sanity/polywrap.yaml

```diff
@@ -0,0 +1,9 @@
+format: 0.5.0
+project:
+  name: ObjectTypes
+  type: wasm/typescript
+source:
+  schema: ./polywrap.graphql
+  module: ./src/index.ts
```

Comment:
NIT: I think we need to (1) make this `module: ./wrap.js` and introduce a new section where we define our codegen directory:
```yaml
bindgen:
  directory: ./src/wrap
  schema: ./polywrap.graphql
  impl: wrapscan.io/polywrap/wrap-typescript-abi-bindgen@1
```

Reply:
I agree with this approach for this PR.

That said, I think we should look into de-escalating the introduction of new config concepts; maybe we could look into how to make these concepts command line arguments or something similar.